docs/proposals/improve-status-in-CAPI-resources.md

```diff
@@ -0,0 +1,940 @@
+
```

Comment:
nit: we should add the date as prefix to the doc (not sure if breaks the open conversations though)